packages/sdk-js/src/GrowthBook.ts

```diff
@@ -1146,8 +1195,34 @@ export class GrowthBook<
     else console.log(msg, ctx);
   }
 
+  public getDeferredTrackingCalls() {
+    return btoa(JSON.stringify(this._deferredTrackingCalls));
```

Comment:
Consider moving encoding stuff out of the getters and setters. We shouldn't be opinionated at this level about network transport

Reply:
If we could provide a service, we could be opinionated in instances of the service rather than the API here